lib/ProductOpener/URL.pm

```diff
@@ -64,8 +64,8 @@ sub subdomain_supports_https {
 	my ($sd) = @_;
 	
 	return $sd unless $sd;
-	return 1 if '*' ~~ @ssl_subdomains;
-	return $sd ~~ @ssl_subdomains;
+	return 1 if grep $_ eq '*', @ssl_subdomains;
+	return grep $_ eq $sd, @ssl_subdomains;
 
```

Comment:
This function is called a lot of times when generating a page, instead of doing a grep, it would be much more efficient to create a hash with the @ssl_subdomains values as keys during initialization of the module, and then use the hash to quickly check it.

That being said, we now have wildcards, so @ssl_subdomains contains only "*"  (at least in production)